airflow-core/src/airflow/models/dag.py

```diff
@@ -2269,6 +2271,18 @@ def get_asset_triggered_next_run_info(self, *, session=NEW_SESSION) -> dict[str,
         # an empty dict as there's no asset info to get. This method should thus return None.
         return get_asset_triggered_next_run_info([self.dag_id], session=session).get(self.dag_id, None)
 
+    @staticmethod
+    @provide_session
+    def get_team_name(dag_id: str, session=NEW_SESSION) -> str | None:
+        """Return the potential team name associated to a Dag."""
```

Comment:
Wording nit: feel free to take it or leave it!
```suggestion
        """Return the team name associated to a Dag or None if it is not owned by a specific team."""
```